packages/cli/lib/__tests__/validators/themeConfigValdiator.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const themeConfigValidator = require('../../validators/marketplaceValidators/themeConfigValidator');
+const { VALIDATION_RESULT } = require('../../validators/constants');
```

Comment:
Instead of having the tests live in `lib`, I'd move to inside `validators`. It is typical for jest tests to be adjacent to code that the tests cover.

Reply:
Moved in ee4a6dd